.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
```

Comment:
Same as https://github.com/reanahub/reana-demo-root6-roofit/pull/42#pullrequestreview-521751819

Reply:
Discussed offline: We keep py2 as the community still uses this version and it good to have it tested.